apps/api/src/app/events/usecases/send-message/send-message-in-app.usecase.ts

```diff
@@ -247,17 +265,17 @@ export class SendMessageInApp extends SendMessageBase {
             events: command.events,
             total_count: command.events?.length,
           },
+          branding: {
+            logo: organization?.branding?.logo,
+            color: organization?.branding?.color || '#f47373',
```

Comment:
When creating an organization we don't brand it by default? Logo for sure not, but I thought the color yes.

Reply:
No we do this trick instead...